modules/AWS/eks/main.tf

```diff
@@ -14,58 +15,65 @@ module "nodegroup_launch_template" {
 
 module "eks" {
   source  = "terraform-aws-modules/eks/aws"
-  version = "17.24.0"
+  version = "18.30.2"
 
   # Configure cluster
-  cluster_version              = "1.21"
-  cluster_name                 = var.cluster_name
-  manage_cluster_iam_resources = true
+  cluster_version = "1.21"
+  cluster_name    = var.cluster_name
+  create_iam_role = true
+  create_cloudwatch_log_group = false
```

Comment:
e2e iam user does not have privileges to create log groups. In future, we can make it configurable as well (if eks logging is a requirement)